windtunnel/windtunnel.py

```diff
@@ -74,59 +148,40 @@ def get_commands(self):
         ]
         return commands
 
-    def simulate(
-        self,
-        object_path: str,
-        wind_speed_ms: float,
-        rotate_z_degrees: float,
-        num_iterations: int,
-        resolution: int,
-        display: bool = False,
-        machine_group_name: Optional[str] = None,
-    ):
-        """Simulates the wind tunnel scenario synchronously.
-
-        Args:
-            object_path: Path to object inserted in the wind tunnel.
-            wind_speed_ms: Velocity of the air flow in m/s.
-            num_iterations: Number of iterations to run the simulation.
-            resolution: Resolution of the simulation grid.
-            display: Whether to display the meshes of object and wind tunnel on
-                the screen, before running the simulation.
-            machine_group_name: Name of the machine group to run simulation on.
+    def display(self):
+        """Display the wind tunnel scenario."""
+        visualizer = WindTunnelVisualizer(**self._walls)
+        if self.object:
+            visualizer.add_mesh(self.object, color="blue", opacity=0.5)
+        visualizer.show()
+
+    def simulate(self,
+                 wind_speed_ms: float,
+                 num_iterations: int,
+                 resolution: int,
+                 machine_group_name: Optional[str] = None,
+                 inputs_base_dir: Optional[str] = "./inductiva_input"):
+        """Runs wind tunnel simulation with specified parameters.
+
+        Parameters:
+            wind_speed_ms (float): Air flow speed in m/s.
+            num_iterations (int): Simulation iteration count.
+            resolution (int): Grid resolution.
+            machine_group_name (Optional[str]): Machine group for simulation.
+            inputs_base_dir (Optional[str]): Input files directory.
```

Comment:
"Local directory where the OpenFOAM inputs files that are constructed by the WindTunnel class will be stored. This can be useful for later reference, and for inspecting exactly what the Inductiva API passed to OpenFOAM. If set to None, nothing will be stored on the local disk (i.e. the program will just use a temp directory that is deleted)."